backend/src/main/java/com/redhat/cloud/notifications/routers/internal/InternalResource.java

```diff
@@ -469,4 +481,29 @@ public Response getDailyDigestTimePreference(@PathParam("orgId") String orgId) {
             return Response.status(NOT_FOUND).build();
         }
     }
+
+    /**
+     * Sends a daily digest command to the engine via a REST request.
+     * @param triggerDailyDigestRequestDto the settings of the digest.
+     */
+    @Consumes(APPLICATION_JSON)
+    @POST
+    @Path("/daily-digest")
+    @RolesAllowed(ConsoleIdentityProvider.RBAC_INTERNAL_USER)
+    public void triggerDailyDigest(@NotNull @Valid final TriggerDailyDigestRequestDto triggerDailyDigestRequestDto) {
+        if (!this.environment.isEnvironmentLocal() && !this.environment.isEnvironmentStage()) {
+            throw new BadRequestException("the daily digests can only be triggered in the stage environment");
+        }
+
+        if (
+            !this.applicationRepository.applicationBundleExists(
+                triggerDailyDigestRequestDto.getApplicationName(),
+                triggerDailyDigestRequestDto.getBundleName()
+            )
+        ) {
+            throw new BadRequestException("unable to find the specified application — bundle combination");
```

Comment:
`—` that's a huge dash😄

Reply:
I tend to use the em dash everywhere! :joy: